pennylane/_qutrit_device.py

```diff
@@ -0,0 +1,390 @@
+            If a list of integers is passed, the circuit evaluations are batched over the list of shots.
+        r_dtype: Real floating point precision type.
+        c_dtype: Complex floating point precision type.
+    """
+
+    # TODO: Update set of supported observables as new observables are added
+    observables = {
+        "Identity",
+    }
+
+    @classmethod
+    def capabilities(cls):
+
+        capabilities = super().capabilities().copy()
+        capabilities.update(model="qutrit")
+        return capabilities
+
+    # Added here for tests to pass (See tests.test_qutrit_device.ExtractStatistics.test_results_no_state())
+    @property
+    def state(self):
+        """Returns the state vector of the circuit prior to measurement.
+
+        .. note::
+
+            Only state vector simulators support this property. Please see the
+            plugin documentation for more details.
+        """
+        raise NotImplementedError
+
+    def statistics(self, observables, shot_range=None, bin_size=None):
+        # Overloading QubitDevice.statistics() as VnEntropy and MutualInfo not yet supported for QutritDevice
+        results = []
+
+        for obs in observables:
+            # Pass instances directly
+            if obs.return_type is Expectation:
+                results.append(self.expval(obs, shot_range=shot_range, bin_size=bin_size))
+
+            elif obs.return_type is Variance:
+                results.append(self.var(obs, shot_range=shot_range, bin_size=bin_size))
+
+            elif obs.return_type is Sample:
+                results.append(self.sample(obs, shot_range=shot_range, bin_size=bin_size))
+
+            elif obs.return_type is Probability:
+                results.append(
+                    self.probability(wires=obs.wires, shot_range=shot_range, bin_size=bin_size)
+                )
+
+            elif obs.return_type is State:
+                if len(observables) > 1:
+                    raise qml.QuantumFunctionError(
+                        "The state or density matrix cannot be returned in combination"
+                        " with other return types"
+                    )
+                if self.wires.labels != tuple(range(self.num_wires)):
+                    raise qml.QuantumFunctionError(
+                        "Returning the state is not supported when using custom wire labels"
+                    )
+                # Check if the state is accessible and decide to return the state or the density
+                # matrix.
+                results.append(self.access_state(wires=obs.wires))
+
+            elif obs.return_type is not None:
+                raise qml.QuantumFunctionError(
+                    f"Unsupported return type specified for observable {obs.name}"
+                )
+
+            # TODO: Add support for MutualInfo and VnEntropy return types. Currently, qml.math is hard coded
```

Comment:
What aspects of `qml.math` are hard-coded?

Reply:
The helper functions in `qml.math` that calculate `VnEntropy` and `MutualInfo` have hardcoded 2's, which is why they can't be used to make those measurements for qutrits yet.